docs/data-sources/pipeline.md

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Compute"
+---
+# databricks_pipeline Data Source
+
+-> **Note** If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../index.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _authentication is not configured for provider_ errors.
+
+Retrieves a name-to-id map of all [Delta Live Tables](https://docs.databricks.com/data-engineering/delta-live-tables/index.html) (DLT) pipelines deployed in a workspace, or those matching the provided search term. 
```

Comment:
if it retrieves all pipelines, then data source should be plural - `databricks_pipelines`

Reply:
data_source and docs updated to reflect this